setup.py

```diff
@@ -14,6 +14,9 @@
             'matplotlib',
             'jpype1==0.6.3',
             'scipy',
+            'pytorch==1.4.0',
+            'catboost==0.23',
+            'xgboost==1.0.2'
```

Comment:
A couple of notes on the dependencies.  First, is it necessary to fix them to specific versions?  This may make it more difficult to install.  I suggest removing the version numbers unless you know that specific versions are required in order for your code to run (the `jpype1==0.6.3` dependency is something that is specifically requested by GNATS).  On Ubuntu 18, I get this error:
```
error: Could not find suitable distribution for Requirement.parse('xgboost==1.0.2sklearn')
```

Secondly, there seems to be a problem with installing `xgboost` in Python 3.6.9 in Ubuntu 18 in general.  I tried removing the version numbers from the setup file, installing manually using `pip install xgboost`, and even building xgboost from the git source.  But I was not able to get it installed.

On this OS, it may be necessary to install an Anaconda Python distribution in order to get this to work.  I can look into that.  However, I want to remove it as a dependency of para-atm if it is not widely available, otherwise others will have this same problem, even if they are trying to use para-atm for something that does not require the aviation risk module.  Removing them as dependencies will mean that users will need to install the packages themselves in order to use your module, which will need to be documented.  This is OK.  You can also add a simple check such as the following to provide a friendly error message:
```python
try:
    import xgboost
except ImportError:
    print('aviation risk module requires installation of xgboost')
    raise
```

Reply:
Got it. Thanks for the comments. I will remove this dependency and let users to make their installation.

Reply:
Yes, please go ahead and remove `pytorch`, `catboost`, and `xgboost` from `setup.py`.  I think the strategy that we will adopt for para-atm will be to only include typical/standard dependencies in `setup.py` (such as numpy, pandas, matplotlib, etc.)  Users that need to use your module can always install the additional dependencies themselves, but those dependencies should be documented somewhere (e.g. in the module-level docstring for `aviationr.py`).